src/openvic-simulation/interface/GUI.cpp

```diff
@@ -6,19 +6,30 @@ using namespace OpenVic;
 using namespace OpenVic::GUI;
 using namespace OpenVic::NodeTools;
 
+Position::Position() : position {} {}
+
+bool Position::_fill_key_map(NodeTools::case_insensitive_key_map_t& key_map) {
+	bool ret = Named::_fill_key_map(key_map);
+	ret &= add_key_map_entry(key_map,
+		"position", ONE_EXACTLY, expect_fvec2(assign_variable_callback(position))
+	);
+	return ret;
+}
+
 Element::Element() : position {}, orientation { orientation_t::UPPER_LEFT } {}
 
 bool Element::_fill_key_map(NodeTools::case_insensitive_key_map_t& key_map, UIManager const& ui_manager) {
 	bool ret = Named::_fill_key_map(key_map, ui_manager);
 	using enum orientation_t;
 	static const string_map_t<orientation_t> orientation_map = {
-		{ "UPPER_LEFT", UPPER_LEFT }, { "LOWER_LEFT", LOWER_LEFT },
+		{ "UPPER_LEFT", UPPER_LEFT }, { "UPPERL_LEFT", UPPER_LEFT }, { "LOWER_LEFT", LOWER_LEFT },
```

Comment:
Typo?
UPPER**L**_LEFT

Reply:
`UPPERL_LEFT` appears in the vanilla `.gui` files where it's treated like `UPPER_LEFT`. It may be a typo that's just defaulted to `UPPER_LEFT`, but adding it as an option here is the simplest way to avoid errors when loading vanilla and still produce the same UI/visual results.

Reply:
`UPPERL_LEFT` appears in the vanilla `.gui` files where it's treated like `UPPER_LEFT`. It may be a typo that's just defaulted to `UPPER_LEFT`, but adding it as an option was the simplest way to avoid errors when loading vanilla and still produce the same UI/visual results.
Tomorrow I can add custom code for parsing orientations that logs a warning for invalid entries and defaults to `UPPER_LEFT`, instead of using `expect_mapped_string` with `UPPERL_LEFT` as an option.